tests/robot-tests/scripts/run_tests_pipeline.py

```diff
@@ -0,0 +1,63 @@
+import argparse
+import subprocess
+
+# NOTE(mark): The slack webhook url, and admin and analyst passwords to access to Admin app are
+# stored in the CI pipeline as secret variables, which means they cannot be accessed as normal
+# environment variables, and instead must be passed as an argument to this script.
+
+
+def run_tests_pipeline():
+    assert args.admin_password, "Provide an admin password with an '--admin-pass PASS' argument"
+    assert args.analyst_password, "Provide an analyst password with an '--analyst-pass PASS' argument"
+    assert args.slack_webhook_url, "Please provide slack webhook URL"
+    assert args.env, "Provide an environment with an '--env ENV' argument"
+    assert args.file, "Provide a file/dir to run with an '--file FILE/DIR' argument"
+
+    valid_environments = ['dev', 'test', 'pre-prod', 'prod', 'ci']
+
+    if args.env not in valid_environments:
+        raise Exception(f"Invalid environment provided: {args.env}. Valid environments: {valid_environments}")
+
+    print('Installing dependencies')
+    subprocess.check_call(['google-chrome-stable', '--version'])
+    subprocess.check_call(["python -m", "pip install --upgrade pip"])
+    subprocess.check_call(["pip install", "pipenv"])
+    subprocess.check_call(["pipenv", "install"])
+
+    # utility to not send slack test report if snapshot tests are being run
+    def should_send_test_reports() -> bool:
+        if args.file == 'tests/general_public/check_snapshots.robot':
+            return False
+        else:
+            return True
```

Comment:
I think with the comment it's still pretty clear what this does if you shorten it down to just the return statement like so

```
def should_send_test_reports() -> bool:
    return args.file != 'tests/general_public/check_snapshots.robot'
```

Reply:
Nice, yeah that looks a lot cleaner